syncopy/tests/test_welch.py

```diff
@@ -362,6 +349,10 @@ def test_welch_rejects_invalid_output(self):
                 with pytest.raises(SPYValueError, match="output"):
                     _ = spy.freqanalysis(cfg, self.adata)
 
+    def teardown_class(cls):
```

Comment:
yes, this we could probably need also elsewhere, maybe put it inside a `helper` module?

Reply:
I moved it to `tests/helpers.py`